src/hera.cpp

```diff
@@ -496,6 +505,21 @@ int hera_set_option(
     hera->metering = strcmp(value, "true") == 0;
     return 1;
   }
+
+  if (strcmp(name, "engine") == 0) {
```

Comment:
I prefer using `std::string` for comparisons, but I'm ok with leaving this as is because of the previous comparisons using `stdcmp()`.

Reply:
@chfast I plan to refactor this in a separate PR.